advanced/sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py

```diff
@@ -202,7 +202,7 @@ def diagnostics_callback(self, msg):
             for motor in self.motors:
                 if motor.motor_name in status.name and self._prefix.replace("/", "") in status.name:
                     for key_values in status.values:
-                        if "Firmware svn revision" in key_values.key:
+                        if "Firmware git revision (server / pic / modified)" in key_values.key:
```

Comment:
this is not that good piece of code in general. Probably it should be something like this key_values.key startswith  "Firmware git revision"